### PR TITLE
use tscn format

### PR DESCRIPTION
**Pull request by c-cesar**
As tscn format is the new default, and more conveniet for git, I converted the old scn files to new format.

### CODE REVIEW

File: device/ui/inventory.tscn [hunk 1]
```diff
@@ -0,0 +1,97 @@
+[gd_scene load_steps=3 format=1]
+
+[ext_resource path="res://globals/inventory.gd" type="Script" id=1]
+[ext_resource path="res://game/inventory_items.scn" type="PackedScene" id=2]
```
Comment (Ranito) [inline device/ui/inventory.tscn]: Fixed dependency:

``[ext_resource path="res://game/inventory_items.tscn" type="PackedScene" id=2]``

File: device/ui/hud.tscn [hunk 2]
```diff
@@ -0,0 +1,50 @@
+[gd_scene load_steps=4 format=1]
+
+[ext_resource path="res://globals/hud.gd" type="Script" id=1]
+[ext_resource path="res://ui/inventory.scn" type="PackedScene" id=2]
```
Comment (Ranito) [inline device/ui/hud.tscn]: This dependency needs fixing.

`[ext_resource path="res://ui/inventory.tscn" type="PackedScene" id=2]`

File: device/ui/hud.tscn [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+[ext_resource path="res://ui/verb_menu.scn" type="PackedScene" id=3]
```
Comment (Ranito) [inline device/ui/hud.tscn]: This dependency needs to be fixed.

`[ext_resource path="res://ui/verb_menu.tscn" type="PackedScene" id=3]`

**Comment by punto-** [comment]
When we made the demo, there was a bug in the tscn format that was failing
to load the lightmaps on the main screens, we should test if that's fixed
in the last stable release first..

**Comment by StraToN** [comment]
@punto- The lightmap bug is still present in current 2.1.3. I'll check if there's an issue related to this.

**Comment by StraToN** [comment]
@c-cesar There are some missing errors in TSCN files you missed. 
Please look at https://github.com/StraToN/escoria/tree/c-cesar-master to update your PR if you wish (else we'll use mine)

We won't pull this until the related bug is not fixed in Godot.

**Comment by StraToN** [comment]
JFYI, I am starting from this PR to adapt current Escoria to Godot 3.

**Pull request closed without merging**